src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+import Big from "big.js";
+
+function operate(numberOne, numberTwo, operation) {
+  let result = 0;
```

Comment:
both numberOne and numberTwo should be wrapped in big.js, here is how you could handle it.
```
const num1 = Big(numberOne)
const num2 = Big(numberTwo)
```